experiments/tester/src/RNTester/TestComponents/RadioGroupTest.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import { RadioButton, RadioGroup } from 'react-native-uifabric';
+// import { RadioButton } from '../../components';
+// import { RadioGroup } from '../../components';
+
+export const RadioGroupTest: React.FunctionComponent<{}> = () => {
+  // Client's example onChange function
+  const onChange = (key: string) => {
+    return;
```

Comment:
Perhaps try calling the `alert()` function so that in the test app, we know it's working.

Reply:
I tested it out, but it tends to get annoying when testing other features such as narrator. I'm going to leave it out for now.